Makefile

```diff
@@ -37,11 +37,14 @@ verify-published-rpms: ## Ensure rpms have been published
 
 # TODO: Uncomment verify-shellcheck once we finish shellchecking the repo.
 #       ref: https://github.com/kubernetes/release/issues/726
-verify: release-tools verify-boilerplate verify-dependencies verify-golangci-lint verify-go-mod #verify-shellcheck ## Runs verification scripts to ensure correct execution
+verify: release-tools verify-boilerplate verify-build verify-dependencies verify-golangci-lint verify-go-mod #verify-shellcheck ## Runs verification scripts to ensure correct execution
 
 verify-boilerplate: ## Runs the file header check
 	./hack/verify-boilerplate.sh
 
+verify-build: ## Builds the project for a chosen set of platforms
+	./hack/verify-build.sh
```

Comment:
I think this target should be added as a PHONY target on [L26](https://github.com/kubernetes/release/blob/bc9c5bf82886807b005a6d7a17cf97d725e63770/Makefile#L26).

Reply:
Fixed :+1: